Create registration.md

jstathas opened the pull request:
Creating 2022 Registration page

jstathas committed:
Create registration.md

osg-cat commented on docs/2022/registration.md:
```diff
@@ -0,0 +1,16 @@
+# OSG All-Hands Meeting 2020 &ndash; Registration
```
2022

osg-cat commented on docs/2022/registration.md:
```diff
@@ -0,0 +1,16 @@
+
+Registration for the OSG All-Hands Meeting 2022 costs **$70**, which includes:
+
+* Attendance Monday through Wednesday, (Thursday and Friday are Virtual only)
```
```suggestion
* Attendance Monday through Wednesday (Thursday and Friday are virtual only)
```

osg-cat commented on docs/2022/registration.md:
```diff
@@ -0,0 +1,16 @@
+Registration for the OSG All-Hands Meeting 2022 costs **$70**, which includes:
+
+* Attendance Monday through Wednesday, (Thursday and Friday are Virtual only)
+* morning snack/light breakfast, afteroon snacks, and beverages throughout
```
```suggestion
* Hot beverage service all day, morning and afternoon snacks
```

osg-cat commented on docs/2022/registration.md:
```diff
@@ -0,0 +1,16 @@
+# OSG All-Hands Meeting 2020 &ndash; Registration
+
```
```suggestion

## Registration for the In-Person Parts

```

osg-cat commented on docs/2022/registration.md:
```diff
@@ -0,0 +1,16 @@
+# OSG All-Hands Meeting 2020 &ndash; Registration
+
+## Costs
```
```suggestion
### Costs for in-person attendance
```

osg-cat commented on docs/2022/registration.md:
```diff
@@ -0,0 +1,16 @@
+* morning snack/light breakfast, afteroon snacks, and beverages throughout
+
+
+## Registration
```
```suggestion
### Registering for in-person attendance
```

osg-cat commented on docs/2022/registration.md:
```diff
@@ -0,0 +1,16 @@
+
+The registration system will be linked from here very soon.
+
+When registering, please indicate which days you plan to attend so that we have
```
We dropped this, right?

osg-cat commented on docs/2022/registration.md:
```diff
@@ -0,0 +1,16 @@
+The registration system will be linked from here very soon.
+
+When registering, please indicate which days you plan to attend so that we have
+accurate counts for catering and so forth.
```
It would be good to include the sentence or two about refunds if OSG cancels here.

jstathas committed:
Update registration.md

jstathas committed:
Update docs/2022/registration.md

Co-authored-by: Tim Cartwright <cat@cs.wisc.edu>

jstathas committed:
Update docs/2022/registration.md

Co-authored-by: Tim Cartwright <cat@cs.wisc.edu>

jstathas committed:
Update docs/2022/registration.md

Co-authored-by: Tim Cartwright <cat@cs.wisc.edu>

jstathas committed:
Update docs/2022/registration.md

Co-authored-by: Tim Cartwright <cat@cs.wisc.edu>

jstathas committed:
Update registration.md

Page refinements - additions about registration deadline and registration refund

jstathas committed:
Update registration.md

Adding clarification about registration forms.

jstathas committed:
Update registration.md

jstathas committed:
Update registration.md

osg-cat committed:
My edits
